pom.xml

```diff
@@ -30,159 +35,51 @@
 		<dependency>
 			<groupId>org.dcsa</groupId>
 			<artifactId>dcsa_event_core</artifactId>
-			<version>${dcsa.events.version}${dcsa.event.tag}${dcsa.artifacttype}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.jacoco</groupId>
```

Comment:
I'd like us to keep the jacoco plugin. It's for the purpose of code coverage to be used later.

Reply:
But you haven't got a module to assemble the reports, which means essentially it does nothing currently